libs/designsystem/src/lib/components/progress-circle/progress-circle-ring.component.scss

```diff
@@ -0,0 +1,23 @@
+@import '../../scss/utils';
+
+.circle {
+  stroke: get-color('semi-light');
+}
+.progress {
+  transition-property: stroke-dashoffset, stroke;
+  transition-duration: #{get-transition-duration('extra-long')};
+  transition-timing-function: #{get-transition-easing('enter-exit')};
+  transform: rotate(-90deg);
+  transform-origin: 50% 50%;
+  stroke: var(--stroke-color);
+}
+
+:host {
+  --stroke-color: #{get-color('success')};
```

Comment:
Consider applying default in component instead of in scss:
` @Input() themeColor: 'success' | 'warning' | 'danger' = 'success';`

Reply:
Moved default to TS as suggested